storage/collector.go

```diff
@@ -35,9 +38,9 @@ func InsertCollector(newCollector Collector) error {
 		return fmt.Errorf("connect error: %q", err)
 	}
 
-	database := client.Database(database)
-	collectorCollection := database.Collection(collector)
-	_, err = collectorCollection.InsertOne(context.TODO(), newCollector)
+	collectorC := client.Database(database).Collection(collectorCollection)
+	setIndexesCollector(collectorC)
```

Comment:
Checar erros

Reply:
Adicionei a busca pelo path por acreditar que o id pode vir a se repetir por geralmente ser a sigla de um órgão. Mas como esse problema não existe hoje, já retirei.

--

A função *Collection()* não retorna erro.
Fiz a checagem da seguinte forma:
`if collectorC == nil {
		return fmt.Errorf("error in retrive collection")
	}`

Checagem para setIndexes foi adicionanda.